src/vochain/vochain.proto

```diff
@@ -200,11 +216,61 @@ message AdminTx {
 }
 
 message RegisterKeyTx {
-        bytes nonce = 1;  // Unique number per vote attempt, so that replay attacks can't reuse this payload
-        bytes processId = 2;  // The process for which the vote is casted
-        Proof proof = 3;  // Franchise proof
-        bytes newKey = 4;   // New key to register
-        string weight = 5;   // Weight to delegate to newKey
+	bytes nonce = 1;  	  // Unique number per vote attempt, so that replay attacks can't reuse this payload
+	bytes processId = 2;  // The process for which the vote is casted
+	Proof proof = 3;  	  // Franchise proof
+	bytes newKey = 4;     // New key to register
+	string weight = 5;    // Weight to delegate to newKey
+}
+
+message MintTokensTx {
+	TxType txtype = 1;
+	uint32 nonce = 2;
+	bytes to = 3;
+	bytes value = 4;
+}
+
+message SendTokensTx {
+	TxType txtype = 1;
+	uint32 nonce = 2;
+	bytes from = 3;
+	bytes to = 4;
+	bytes value = 5;
+}
+
+message SetTransactionCostsTx {
+	TxType txtype = 1;
+	uint32 nonce = 2;
+	uint64 value = 3;
+}
+
+message SetAccountInfoTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	string infoURI = 3;
+}
+
+message SetAccountDelegateTx {
```

Comment:
How are we gonna identify if the tx sender wants to add or delete a delegate? I think it's worth to add a bool (`deleteExisting`) or something like that.

Reply:
Using the proper `TxType`:
```
ADD_DELEGATE_FOR_ACCOUNT = 18;
DEL_DELEGATE_FOR_ACCOUNT = 19;
```
I also thought about the bool but we can skip it using the TxType

Reply:
right